src/libbluechi/test/common/network/assemble_tcp_address.c

```diff
@@ -69,7 +69,8 @@ bool test_assemble_tcp_address_v6(const struct sockaddr_in6 *input, const char *
 
 bool test_assemble_tcp_address(const struct sockaddr_in *input, const char *expectedResult) {
         _cleanup_free_ char *result = assemble_tcp_address(input);
-        if ((result == NULL && expectedResult != NULL) || (result != NULL && !streq(result, expectedResult))) {
+        if ((result == NULL && expectedResult != NULL) || (result != NULL && expectedResult == NULL) ||
+            (result != NULL && !streq(result, expectedResult))) {
```

Comment:
Wouldn't having one condition block per line make the code more readable?

```
        if ((result == NULL && expectedResult != NULL) || 
            (result != NULL && expectedResult == NULL) ||
            (result != NULL && !streq(result, expectedResult))) {
```

Reply:
I agree that the suggested formatting would be more readable, but, unfortunately, the formatting is enforced like this :-/